frontend_tests/run-casper

```diff
@@ -39,73 +39,85 @@ parser.add_option('--remote-debug',
                   default=False)
 (options, args) = parser.parse_args()
 
-test_files = []
```

Comment:
Can you split the refactoring part of this (adding the `server` argument and moving around the code) into its own commit (that we can merge before the changes that actually add the new test?)